components/Message/Message.test.js

```diff
@@ -0,0 +1,66 @@
+import Message from './Message'
+
+describe('Single message component tests', ()=> {
+  const message1 = {
+    sender: 'TestMan',
+    recipient: '123123124',
+    subject: 'Testing messages',
+    body: 'This is a test message',
+    createdAt: 1615934423695,
+  }
+
+
+  it('renders', () => {
+    shallow(<Message />)
+  })
+
+  it('accepts a message object as a prop', () => {
+    const component = mount(<Message message={message1} />)
```

Comment:
Mounting tends to be an ill practice when it comes to testing. `shallow` should cover all cases that you need because you shouldn't be worrying about testing nested child components cause they will have their own tests.